frontend/src/infra/MyProvider.js

```diff
@@ -40,40 +29,48 @@ class MyProvider extends Component {
     });
     const req = { creatorID: playerId };
     axios
-      .post(SERVER_ADDRESS + ":" + SERVER_PORT + "/room/create", req)
-      .then(res => {
-        const { roomID, a, creationDate } = res.data;
+      .post(`${SERVER_ADDRESS}:${SERVER_PORT}/room/create`, req)
+      .then((res) => {
+        console.log(res.data);
+        const { roomID, a, creationDate, socketPort } = res.data;
+        createSocketAndListen(socketPort, "NewPlayer", (data) => {
+          this.setState({
+            playerList: data.roomPlayers,
+          });
+        });
         this.setState({
           currentRoom: roomID,
-          roomCreationDate: creationDate
+          roomCreationDate: creationDate,
         });
       });
   };
-
-  joinGame = roomId => {
+  joinGame = (roomId) => {
     const playerId = Shortid.generate();
     this.setState({
       phase: PHASE.WAITING_ROOM,
       currentRoom: roomId,
-      playerId: playerId
+      playerId: playerId,
     });
   };
 
   onNameEntered = (name) => {
```

Comment:
onNameEntered where?

Reply:
changed to onPlayerRegisterToRoom 1f6189a